MiniAn/minian_run.py

```diff
@@ -91,12 +91,12 @@
             params[params_.replace('params_','')+'-'+key] = value
 
 params_LocalCluster = dict(
-    n_workers=int(os.getenv("MINIAN_NWORKERS", 4)),
-    memory_limit="3GB",
-    resources={"MEM": 1},
-    threads_per_worker=2,
-    dashboard_address=":8787",
-    local_directory=dpath
+    n_workers = 4,
```

Comment:
How can we decide the number of workers?

Reply:
before it was `n_workers=int(os.getenv("MINIAN_NWORKERS", 4))` and on what I saw the MINIAN_NWORKERS environment variable make no sense when MiniAn is not "installed". Also I couldn't find how this environment variable is defined. So I removed it and just set it at 4.

I can revert that if you want

Reply:
> It's not about wanting haha. I'm trying to get sense out of it, so of course, before going on and searching on my own, I'm asking you, first, because you're working on it. So as I understood, we don't know why it's 4 and why it's this environment. Did you check their jupyter notebook and/or paper?

Indeed, We don't know why 4. I assume it's because a lot of CPU have 4 cores. In the pipeline it is set like that `int(os.getenv("MINIAN_NWORKERS", 4))`

I didn't check the paper.

Reply:
Quoting from the above link - one way to memory issue would be decreasing the chunk size rather than the memory_limit or number of workers-
"MiniAn try to find the best chunk size for you using [get_optimal_chk]under the loading videos and visualization part of the pipeline. The default is to produce chunks that use around 256MB of memory (controlled by csize argument), which is roughly 1/10 of the default memory_limit for each worker. Hence, if you find your workers struggling with memory, and you don’t have more physical RAM to spare to increase memory_limit, you may consider decreasing the chunk size."

Reply:
I didn't saw this page. That is interesting 

I could set the number of worker equal or half of the number of total computer core.

Also for the csize I could check to see if setting it lower would change something. But like what would be the rule for setting the value :/ like 1/10 is pretty low already

Reply:
Interesting look like setting `csize = 100` remove the memory overload on saving the datasets....